addons/website_slides_survey/controllers/main.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.website_slides.controllers.main import WebsiteSlides
+from odoo.addons.survey.controllers.main import Survey
+
+from odoo import http
+from odoo.http import request
+
+
+class WebsiteSlides(WebsiteSlides):
```

Comment:
Can we split controller into separated file ? like survery.py, slides.py ....

Reply:
Split and good to go!